app/src/hooks/cb_api/useMarkingApi.tsx

```diff
@@ -1,8 +1,47 @@
 import { AxiosInstance } from 'axios';
 import { ICreateCritterMarking } from 'features/surveys/view/survey-animals/animal';
 import { IMarkingResponse } from 'interfaces/useCritterApi.interface';
+import {
+  IMarkingBodyLocationResponse,
+  IMarkingColourOption,
+  IMarkingTypeResponse
+} from 'interfaces/useMarkingApi.interface';
 
 const useMarkingApi = (axios: AxiosInstance) => {
+  /**
+   * Get possible marking types
+   *
+   * @async
+   * @returns {Promise<IMarkingTypeResponse>} - The created marking.
+   */
+  const getMarkingTypeOptions = async (): Promise<IMarkingTypeResponse[]> => {
+    const { data } = await axios.get(`api/critterbase/lookups/marking-types`);
+    return data;
+  };
+
+  /**
+   * Get possible marking body locations for a given species
+   *
+   * @async
+   * @param {number} tsn
+   * @returns {Promise<IMarkingTypeResponse>} - The created marking.
+   */
+  const getMarkingBodyLocationOptions = async (tsn: number): Promise<IMarkingBodyLocationResponse[]> => {
```

Comment:
maybe rename to getTaxonMarkingBodyLocationOptions?

Reply:
Renamed